ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.7.yaml

```diff
@@ -138,6 +138,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ovn
+- as: e2e-kubelet-density-step-registry
```

Comment:
did you mean to add this to the MCO repo tests?

Reply:
Yeah, have to double check with some team mates. For now I'm just only doing some tests

Reply:
The MCO repo changes don't generally affect kubelet performance. I feel like this should be added somewhere else, once its ready